gslib/commands/acl.py

```diff
@@ -340,6 +343,41 @@ class AclCommand(Command):
       },
   )
 
+  def get_gcloud_storage_args(self):
+    _CannedGcsAcl = ['private', 'public-read', 'project-private',
+                    'public-read-write', 'authenticated-read',
+                    'bucket-owner-read', 'bucket-owner-full-control']
+    if self.args[0] == 'get':
+      if StorageUrlFromString(self.args[1]).IsObject():
+        command_group = 'objects'
+      elif StorageUrlFromString(self.args[1]).IsBucket():
```

Comment:
`command_group` will be trigger an undeclared variable error if the arg isn't a bucket or object